orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/ContinueParentStageHandler.kt

```diff
@@ -63,18 +69,7 @@ class ContinueParentStageHandler(
     if (firstTask.status == NOT_STARTED) {
       queue.push(StartTask(this, firstTask))
     } else {
-      log.warn("Ignoring $messageType for ${id} as tasks are already running")
-    }
-  }
-
-  private fun Stage.runAfterStages() {
```

Comment:
Why was this deleted?

Reply:
Don't know, this was in the original revert. It's not used.